src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/OpprettBehandlingValg.tsx

```diff
@@ -92,7 +92,9 @@ const OpprettBehandlingValg: React.FC<IProps> = ({
         : minimalFagsak.status !== FagsakStatus.LØPENDE && kanOppretteBehandling;
     const kanOppretteRevurdering = !minimalFagsak
         ? false
-        : minimalFagsak.behandlinger.length > 0 && kanOppretteBehandling;
+        : minimalFagsak.behandlinger.length > 0 &&
+          kanOppretteBehandling &&
+          hentAktivBehandlingPåMinimalFagsak(minimalFagsak) !== undefined;
```

Comment:
Når er behandlinger aktive?
Hva hvis du avslutter en behandling og så starter en ny? Var der forrige fortsatt aktiv da?

Eller skal det være førstegangsbehandling da?

Reply:
Gode spørsmål 😅 Selv liter jeg med terminologien. Min hypotese:
Hvis det ikke finnes noen behandlinger med pågående utbetalinger, så skal det ikke være mulig å opprette revurdering. Men hvis det derimot finnes fullført behandling, og utbetalinger kjører, så skal man kunne opprette en revurdering. aktiv betyr at det utbetalinger er pågående.

Men stor sjanse for at jeg tar feil her. Trenger nok @henninghaakonsen  til å dobbelsjekke her...

Tenkte også å få Eivind til å teste

Reply:
Hmm lurer på om det kanskje skulle vært en sjekk om aktiv behandling har status LØPENDE...

Reply:
Men da blir det uansett rart om vi det ikke lenger er løpende. Si du får et barn og barnetrygd for det barnet. Så slutter barnetrygden 18 år senere. Om du da får et barn etter det igjen vil du ikke ha neon aktiv behandling eller noen løpende barnetrygd, men ville tro det fremdeles skal kunne opprettes en revurdering?